test/test_inverse.py

```diff
@@ -543,7 +582,8 @@ def test_KFAC_inverse_save_and_load_state_dict():
     inv_kfac = KFACInverseLinearOperator(
         kfac, damping=1e-2, retry_double_precision=False, cache=cache, **kwargs
     )
-    _ = inv_kfac @ torch.eye(kfac.shape[1])  # to trigger inverse computation
+    # trigger inverse computation and maybe caching
+    _ = inv_kfac @ torch.eye(kfac.shape[1])
```

Comment:
Shouldn't we follow the same pattern as in the function below (`test_KFAC_inverse_from_state_dict`)? I.e., define `test_vec` and `test_mvp` and then not delete the line below (I will comment).

Reply:
I addressed this and now compute the matrix representation before saving and after loading, then compare them.